app/src/full/java/io/homeassistant/companion/android/settings/wear/views/SettingsWearLandingView.kt

```diff
@@ -35,44 +40,51 @@ fun SettingWearLandingView(
+    ) { padding ->
         Column(
             modifier = Modifier
                 .fillMaxWidth()
-                .padding(all = 16.dp)
+                .padding(padding)
         ) {
-            Text(
-                text = stringResource(id = commonR.string.manage_wear_device, deviceName),
-                textAlign = TextAlign.Center
-            )
+            Row(
+                modifier = Modifier
+                    .height(48.dp)
+                    .padding(start = 72.dp, end = 16.dp),
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Text(
+                    text = stringResource(id = commonR.string.manage_wear_device, deviceName),
+                    style = MaterialTheme.typography.body2,
+                    fontWeight = FontWeight.Bold,
+                    color = MaterialTheme.colors.primary
+                )
+            }
             when {
                 !hasData -> {
                     CircularProgressIndicator(modifier = Modifier.align(Alignment.CenterHorizontally))
                 }
                 isAuthed -> {
-                    Button(
-                        onClick = navigateFavorites,
-                        modifier = Modifier
-                            .fillMaxWidth()
-                            .padding(top = 10.dp)
-                    ) {
-                        Text(stringResource(commonR.string.set_favorites_on_device))
-                    }
-                    Button(
-                        onClick = navigateTemplateTile,
-                        modifier = Modifier
-                            .fillMaxWidth()
-                            .padding(top = 10.dp)
-                    ) {
-                        Text(stringResource(commonR.string.template_tile))
-                    }
+                    SettingsRow(
+                        primaryText = stringResource(commonR.string.favorite),
+                        secondaryText = stringResource(commonR.string.set_favorites_on_device),
+                        mdiIcon = CommunityMaterial.Icon3.cmd_star,
+                        enabled = true,
+                        onSensorClicked = navigateFavorites
+                    )
+                    SettingsRow(
+                        primaryText = stringResource(commonR.string.template_tile),
+                        secondaryText = stringResource(commonR.string.template_tile_set_on_watch),
+                        mdiIcon = CommunityMaterial.Icon3.cmd_text_box,
+                        enabled = true,
+                        onSensorClicked = navigateTemplateTile
+                    )
                 }
                 else -> {
                     Button(
                         onClick = loginWearOs,
                         modifier = Modifier
                             .fillMaxWidth()
-                            .padding(top = 10.dp)
+                            .padding(top = 10.dp, start = 16.dp, end = 16.dp)
```

Comment:
The full width button looks a bit out of place... Did you consider changing it to use a 'normal' settings row as well? (I haven't tried yet)
![Wear settings screen with a full width 'Login Wear OS device' button](https://user-images.githubusercontent.com/8148535/187778799-28fd4125-a4a2-4366-9e18-fb3f8f78f1af.png)

Reply:
I was debating this a bit internally. I like that the button stands out, since it's not a normal settings item. But on the other hand it doesn't fit into the general layout very well. So I'll take suggestions ;)

Reply:
Good points. After testing a normal row and aligning the button with the text it still looks a bit weird to me, so no suggestions for now :)

Reply:
Haha okay, then I'll leave it for now!